src/commands/lambda/__tests__/functions/instrument.part2.test.ts

```diff
@@ -133,7 +133,7 @@ describe('instrument', () => {
     })
 
     test('calculates an update request with a lambda library, extension, and DATADOG_API_KEY', async () => {
-      process.env[CI_API_KEY_ENV_VAR] = '1234'
+      process.env[CI_API_KEY_ENV_VAR] = '1234a'
```

Comment:
Why was this changed?

Reply:
Since masking logic doesn't apply to numbers I added a letter to this API key so unit tests reflected the expected masking behavior

Reply:
Ah I see what you mean. In this case the snapshot shows the DD_API_KEY as is because the update request isn't being printed to the console like it is in the code below. I did a find/replace for all of the instances where the `DD_API_KEY` was `1234` and replaced it with `1234a` so I caught a few where that change wasn't needed for the masking changes.

![Image 2023-07-06 at 2 25 24 PM](https://github.com/DataDog/datadog-ci/assets/35278470/84e9e94d-a737-456f-ac4e-7b01b7d2fd86)

Reply:
Resolved here: https://github.com/DataDog/datadog-ci/pull/961#discussion_r1257300524